config/sitemap.rb

```diff
@@ -2,39 +2,58 @@
 
 SitemapGenerator::Sitemap.create do
 
-  Channel.sitemappable.each do |channel|
+  unless Note.publishable.listable.empty?
+    add notes_path
+
+    feature_indices = []
+
+    Note.listable.publishable.order('feature DESC, weight ASC, feature_id DESC') .each do |note|
```

Comment:
Line is too long. [96/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.